projects/canopy/src/lib/card/card.component.ts

```diff
@@ -60,6 +84,30 @@ export class LgCardComponent implements AfterContentInit, OnDestroy {
     }
   }
 
+  toggleClasses(newClasses: Array<string>, oldClasses: Array<string>): Array<string> {
+    if (oldClasses.length) {
```

Comment:
why this `if` here? i think it's not necessary?

Reply:
i think it wouldn't do that anyway? it would try to loop through it, have 0 things to loop through, and just skip it, right?
there's also a similar `forEach` below this that's treated differently

Reply:
You are correct that it would try to loop through it, have 0 things to loop through, and just skip it, but I wanted to stop before that if its empty, and the 'forEach' would not even try to loop through it.
The second ``forEach`` is treated differently because there is no chance of it being empty, because if it got to that stage then it means the 'orientation' Input got populated hence having the ``newClasses`` so for this one there was no need to check if its empty or not because it can't be empty at this stage, but the ``oldClasses`` array can be empty and that is why I check it before trying the ``forEach`` on it.

For me it made sense like this but can remove the 'if' statement as I don't feel that strong about it in this scenario.

Reply:
seems that i misunderstood what `forEach` was doing under the hood and it's much slower without the `if` around it